src/script/main/app.ts

```diff
@@ -128,6 +128,7 @@ import {Core} from '../service/CoreSingleton';
 
 function doRedirect(signOutReason: SIGN_OUT_REASON) {
   let url = `/auth/${location.search}`;
+  localStorage.setItem('loginredirect', location.hash);
```

Comment:
isn't it better to have the "loginredirect" string as a static variable on App class? like we have in TeamRepository? please also consider changing it to "LOGIN_REDIRECT"
<img width="907" alt="image" src="https://user-images.githubusercontent.com/63250054/153597306-7425fb4a-2e30-4672-86aa-47434d47fae1.png">

Reply:
this function is outside of the class though. im fine saving it to a variable but probably it should be in a different spot

Reply:
done